tests/DoctrineTest.php

```diff
@@ -124,16 +124,14 @@ public function run()
 
         //show help text
         if (isset($options['help'])) {
-            $availableGroups = sort(array_keys($this->groups));
```

Comment:
`sort` doesn't return a new array (IDE was complaining about using a bool in `implode` below).